firecrown/models/cluster/recipes/murata_binned_spec_z_deltasigma.py

```diff
@@ -0,0 +1,227 @@
+"""Module for defining the classes used in the MurataBinnedSpecZ cluster recipe."""
+
+# pylint: disable=duplicate-code
```

Comment:
Both recipes have a huge overlap. Since we will change this after, I believe we can ignore it for now. If you guys do not agree, I can create another parent class from MurataBinned